app/views/users/show.html.erb

```diff
@@ -0,0 +1,25 @@
+<%= stylesheet_link_tag 'users' %>
+<div>
+    <%= render "shared/user", user: @user %>
+    
+   <div class="bio">
+    <h2>Bio</h2>
+    <%= @user.bio%>
+    </div>
+    
+    <%@user_posts.each_with_index do |post, index|%>
+    <div class="post">
+    <h2>post <%=index+1%></h2>
```

Comment:
- [ ] Clicking on each post title takes you to their index page, please 🙏🏽 add the link for a better UX.